examples/logging/README.md

```diff
@@ -0,0 +1,340 @@
+# DeepSparse Logging
+
+DeepSparse Logging provides operational teams with access to the telemetry needed to monitor a deployment. 
+
+For users seeking to put ML into production, these data are the raw materials that underpin the monitoring processes needed to create a system with consistently fresh and accurate predictions.
+
+<p align="center">
+     <img src="continual-learning.png"
+          alt="Continual Learning Diagram"
+          width="50%"
+     />
+</p>
+
+There are many types of downstream monitoring tasks that ML teams may want to perform. Some are easier to do (such as looking at system performance) and some are harder (looking at model accuracy requires manually labeling some data afterwards). Examples include:
+- **System performance:** what is the latency/throughput of a query?
+- **Data quality:** is there an error in the model pipeline?
+- **Data distribution shift:** are the inputs and outputs from the system expected?
+- **Model accuracy:** what is the accuracy of the predictions vs human-labeled?
+
+DeepSparse Logging is designed to provide maximum flexibility for users to extract whatever data is needed from a production inference pipeline into the logging system of their choice. 
+
+## Metrics 
+DeepSparse Logging provides access to two types of metrics:
+- **System Logging Metrics** give operations teams access to granual performance metrics, diagnosing and isolating deployment system health. Examples include CPU utilization and query latency.
+     - See [below - UPDATE LINK](system-logging-metrics) for a full list of available metrics.
+
+- **Data Logging Metrics** give ML teams access to data at each stage of an ML pipeline, supporting downsteam tasks like measuring accuracy and data drift. Examples include raw inputs and projections thereof such as mean pixel value.
+     - See [below - UPDATE LINK](data-logging-functions) for a full list of built-in functions and how to create a custom function
+
+## Configuration
+DeepSparse Logging is configured via YAML files.
+
+**System Logging** is *enabled* by default and all metrics are pre-defined. It can be disabled 
+globally or at the group level by adding the key-value pairs with `on` or `off` status.
+        
+**Data Logging** is *disabled* by default. A YAML configuration file is used to specify which data or functions thereof to log. 
```

Comment:
We start talking about the functions here, I think it is appropriate to have introduced the metric functions at this point.